index.js

```diff
@@ -0,0 +1,79 @@
+const express = require("express");
+const cors = require("cors");
+const session = require("express-session");
+const KnexSessionStore = require("connect-session-knex")(session);
+const bcrypt = require("bcryptjs"); // *************************** added package and required it here
+
+const db = require("./database/dbConfig.js");
+
+const server = express();
+
+const sessionConfig = {
+  name: "monkey",
+  secret: "asfjaofuwruq04r3oj;ljg049fjq30j4jlajg40j40tjojasl;kjg",
```

Comment:
You want to try and store secrets in env variable so it is not accessible for everyone.